source/installation/install-software.rst

```diff
@@ -53,20 +60,21 @@ Some operating systems use `Software Collections`_ to satisfy these.
 
    .. tab:: RHEL 8
 
-      .. warning::
-
-         You may also need to enable the *Optional* channel and
-         attach a subscription providing access to RHSCL to be able to use this
-         repository.
-
       .. code-block:: sh
 
-         sudo dnf config-manager --set-enabled powertools
```

Comment:
I just want to confirm RHEL 9 doesn't need `powertools` for ... I want to say lua-posix?

Reply:
Correct, it uses crb which is Code Ready Builder which is what RHEL8/RHEL9 both called the repo but for some reason downstream CentOS 8 called it powertools. Now downstream calls it crb.